mavis/tools.py

```diff
@@ -478,15 +478,22 @@ def _convert_tool_output(input_file, file_type=SUPPORTED_TOOL.MAVIS, stranded=Fa
                 'normalized_RD',
                 'e-val1', 'e-val2', 'e-val3', 'e-val4', 'q0'
             ])
-    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ]:
+    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ, SUPPORTED_TOOL.STRELKA]:
         rows = []
         vfile = VariantFile(input_file)
         try:
             vfile.header.info.add('END', number=1, type='Integer', description='End of the interval')
         except ValueError:
             pass
         for vcf_record in vfile.fetch():
-            rows.extend(_parse_vcf_record(vcf_record, log=log))
+            try:
+                rows.extend(_parse_vcf_record(vcf_record, log=log))
+            except Exception as err:
```

Comment:
Would be better if you specified the exact error. Do you know what error is thrown here? Or could there be multiple errors? Or do you simply not care what the error is?

Reply:
I'm not sure what error is thrown, I think there could be multiple errors but we just wanted to ignore those lines. 

The reason we added this was because there were a few lines from Strelka that were poorly formated.